nominal/nominal.py

```diff
@@ -279,9 +283,12 @@ def create_run(
     conn = get_default_client()
     return conn.create_run(
         name,
-        start=ts._SecondsNanos.from_flexible(start).to_nanoseconds(),
-        end=None if end is None else ts._SecondsNanos.from_flexible(end).to_nanoseconds(),
+        start=start,
+        end=end,
```

Comment:
```suggestion
        start=ts._SecondsNanos.from_flexible(start).to_nanoseconds(),
        end=None if end is None else ts._SecondsNanos.from_flexible(end).to_nanoseconds(),
```

Reply:
keeping this so the types remain a bit more restricted in core.py, which is intentional